tony-core/src/main/java/com/linkedin/tony/events/HistoryEvent.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2018 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.tony.events;
+
+import org.apache.hadoop.classification.InterfaceAudience;
+import org.apache.hadoop.classification.InterfaceStability;
+
+/**
+ * Interface for event wrapper classes.  Implementations each wrap an
+ * Avro-generated class, adding constructors and accessor methods.
+ */
+@InterfaceAudience.Private
+@InterfaceStability.Unstable
+public interface HistoryEvent {
```

Comment:
Let's also add a `long getTimestamp()` method

Reply:
I'm going to include this in the next PR where I work on the reading part of the history files.